app/Http/Controllers/HistoryController.php

```diff
@@ -8,7 +8,7 @@
 
 class HistoryController extends Controller
 {
-    public function index($id){
+    public function index($id) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->